### PR TITLE
Cross storage class clone /restore

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
https://github.com/openshift/console/assets/6670284/07c937ba-9397-4129-82e1-c1f451f5ff59

**Comment by TimothyAsirJeyasing** [comment]
UX link: https://www.sketch.com/s/63bab1bd-b982-433c-80c7-8d3368a29652/a/v87RmbM

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar Please review

File: frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx [hunk 1]
```diff
@@ -57,23 +63,15 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
...
-  const [response, error, loading] = usePrometheusPoll({
-    endpoint: PrometheusEndpoint.QUERY,
-    query: pvcUsedCapacityQuery,
-    namespace,
-  });
-  const pvcUsedCapacityQueryResult: DataPoint[] = getInstantVectorStats(
-    response,
-    null,
-    humanizeBinaryBytes,
-  );
```
Comment (SanjalKatiyar) [inline frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx:75]: plz don't remove anything like this...

**Comment by SanjalKatiyar** [comment]
this is same provisioner clone/restore, need to add this filter in both modals...

**Comment by SanjalKatiyar** [comment]
> /hold

not needed in 4.15 release...

**Pull request closed without merging**